src/Cache/Cache.php

```diff
@@ -168,7 +168,18 @@ protected static function _buildEngine($name)
         }
 
         $config = static::$_config[$name];
-        $registry->load($name, $config);
+
+        try {
+            $registry->load($name, $config);
+        } catch (RuntimeException $e) {
+            if (!array_key_exists('fallback', $config)) {
```

Comment:
We could use `trigger_error` and rely on a logging being hooked up to an error logger / SAPI logger.